docs/src/pages/quasar-cli/developing-ssr/deploying.md

```diff
@@ -32,3 +32,70 @@ After installing PM2 on your server, your npm start script can look like this in
   "start": "pm2 start index.js"
 }
 ```
+
+## Production scenario
+For serving our Quasar SSR web app in the real world we should consider using Nginx as reverse proxy for leveraging these advantages:
+- Faster static file serving
+- SSL offloading
+- Error pages handling
+- Compression
+- Proxy WebSocket Connections
+- Caching
+
+You may have to consider the scope of your web app, the average number of requests and more other things specific to the reliability of your services.
```

Comment:
I'm not sure this sentence adds a lot of value.  Instead I would include a link to the [Nginx reverse proxy documentation](https://docs.nginx.com/nginx/admin-guide/web-server/reverse-proxy/).

Something like
"The following example demonstrates using both Nginx and Express for serving an SSR Quasar web app.  It is recommend to review the [Nginx documentation](https://docs.nginx.com/nginx/admin-guide/web-server/reverse-proxy/) to better understand the different parameters before using in a production scenario."